vtex/faststore
# chore: move generate graphql to cli

## What's the purpose of this pull request?

This PR fixes an issue where the generate.sh file wouldn't run on Windows systems.

## How it works?

The generate.sh script has been ported to the FastStore CLI. FastStore CLI is now a dev dependency of FastStore Core, which allows it to use its commands. The schema generation and type/optimizations generation is executed through the CLI now. This also allows for better code maintainability, since it's now written in typescript.

## How to test it?

Will provide a deploy preview. Run dev, build and generate commands on the `api-extension-setup` branch and see if the changes made in the GraphQL layer take effect.

To test on FastStore Core, simply develop or build the store.

### Starters Deploy Preview

https://sfj-009e55d--starter.preview.vtex.app/oculus-vr-headset-99988210/p
